docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -876,6 +876,83 @@ To retrieve the information about the data files of the Iceberg table ``test_tab
 
 
 
+The output of the query has the following columns:
+
+.. list-table:: Files columns
+  :widths: 25, 30, 45
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``content``
+    - ``integer``
+    - Type of content stored in the file.
+      The supported content types in Iceberg are:
+
+      * ``DATA(0)``
+      * ``POSITION_DELETES(1)``
+      * ``EQUALITY_DELETES(2)``
+  * - ``file_path``
+    - ``varchar``
+    - The data file location
+  * - ``file_format``
+    - ``varchar``
+    - The format of the data file
+  * - ``record_count``
+    - ``bigint``
+    - The number of entries contained in the data file
+  * - ``file_size_in_bytes``
+    - ``bigint``
+    - The data file size
+  * - ``column_sizes``
+    - ``map(integer, bigint)``
+    - Mapping between the Iceberg column ID and its corresponding size in the file
```

Comment:
Just as a heads-up https://github.com/trinodb/trino/pull/13026 is related to this PR (depending on which of the PRs lands first).

Reply:
I think it would be better to merge #13026 first.

Reply:
@ebyhr a request, can we merge this, please? We also want to use the `$all_files` table.
Also, this PR is pending for some time and we can upgrade the https://github.com/trinodb/trino/pull/13026 once this is merged. thanks!

Reply:
@ebyhr is it ok now if we can merge this PR now, please!
As #13026 is going on, and we can update #13026 once this one is merge.